Dockerfile

```
@@ -32,9 +32,13 @@ WORKDIR /staging
 # Copy main executable to staging area
 RUN cp "$(swift build --package-path /build -c release --show-bin-path)/Run" ./
 
-# Uncomment the next line if you need to load resources from the `Public` directory.
+{{#leaf}}
+# Copy any resouces from the public directory and views directory
+RUN mv /build/Public ./Public && chmod -R a-w ./Public
+RUN mv /build/Resources ./Resources && chmod -R a-w ./Resources
```

I strongly suggest either leaving the `Public` line commented out or adding an `Public/` directory with a `.gitkeep` to the `manifest.yml` when Leaf is enabled, otherwise the resulting `Dockerfile` will not work out of the box:
```
Step 10/20 : RUN mv /build/Public ./Public && chmod -R a-w ./Public
 ---> Running in 92c18b080c6e
mv: cannot stat '/build/Public': No such file or directory
The command '/bin/sh -c mv /build/Public ./Public && chmod -R a-w ./Public' returned a non-zero code: 1
```

Or even just check the directories for existence before operating on them:

```suggestion
RUN [ -d /build/Public ] && { mv /build/Public ./Public && chmod -R a-w ./Public; } || true
RUN [ -d /build/Resources ] && { mv /build/Resources ./Resources && chmod -R a-w ./Resources; } || true
```

---

(Granted, using such a relatively complex Bash-ism - even if I dud spell it the old `/bin/sh` way to make Ubuntu happy - as a multi-command ternary might be a confusing thing to find in a `Dockerfile` for most people... In particular, the `{ pipeline ; }` syntax for not spawning subshells doesn't seem to be widely known/obvious. I don't mean to suggest this is the only or even "best" way to write an existence check 🤷‍♀️; I just can't think of anything more readable that isn't also too painfully verbose for the purpose at the same time.)

---

Opted for a compromise - always copy the `Resources` directory since we include it with Leaf, but only copy the `Public` directory if it exists